File: unit-06-kvanbortel/src/main/java/unit06/Hello.java
package unit06;

import java.util.Scanner;

public class Hello {

    public static void helloYou() {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter your name: ");
        String name = scanner.nextLine();
        System.out.println("Hello, " + name + "!");
        scanner.close();
    }
    
    public static void main(String[] args) {
        helloYou();
    }
}

File: unit-01-kvanbortel/hello.py
"""
This program prints "Hello, World!" and then says hello to the name you enter.
Author: Kayla Van Bortel
"""


def hello_world():
    """
    Prints "Hello, World!"
    """
    print("Hello, World!")  # print "Hello, world!"


def hello_you():
    """
    Prints a personalized hello message
    """
    name = input('Enter your name: ')  # get user's name
    print("Hello,", name)  # print personalized message


def main():
    hello_world()
    hello_you()


main()
